worlds/ladx/ItemIconGuessing.py

```diff
@@ -0,0 +1,483 @@
+    'BIGKEYIP': 'NIGHTMARE_KEY',
+    'BIGKEYMM': 'NIGHTMARE_KEY',
+    'BIGKEYPD': 'NIGHTMARE_KEY',
+    'BIGKEYSP': 'NIGHTMARE_KEY',
+    'BIGKEYSW': 'NIGHTMARE_KEY',
+    'BIGKEYTH': 'NIGHTMARE_KEY',
+    'BIGKEYTR': 'NIGHTMARE_KEY',
+    'BIGKEYTT': 'NIGHTMARE_KEY',
+    'BLUEBOOMERANG': 'BOOMERANG',
+    'BOMBUPGRADE10': 'MAX_BOMBS_UPGRADE',
+    'BOMBUPGRADE5': 'MAX_BOMBS_UPGRADE',
+    'BOTTLEWITHBEE': 'MEDICINE',
+    'BOTTLEWITHBLUEPOTION': 'MEDICINE',
+    'BOTTLEWITHFAIRY': 'MEDICINE',
+    'BOTTLEWITHGOLDBEE': 'MEDICINE',
+    'BOTTLEWITHGREENPOTION': 'MEDICINE',
+    'BOTTLEWITHREDPOTION': 'MEDICINE',
+    'BYRNA': 'MAGIC_ROD',
+    'COMPASSDP': 'COMPASS',
+    'COMPASSEP': 'COMPASS',
+    'COMPASSGT': 'COMPASS',
+    'COMPASSIP': 'COMPASS',
+    'COMPASSMM': 'COMPASS',
+    'COMPASSPD': 'COMPASS',
+    'COMPASSSP': 'COMPASS',
+    'COMPASSSW': 'COMPASS',
+    'COMPASSTH': 'COMPASS',
+    'COMPASSTR': 'COMPASS',
+    'COMPASSTT': 'COMPASS',
+    'ETANK': 'HEART_CONTAINER',
+    'FIFTYRUPEES': 'RUPEES_50',
+    'FIREROD': 'MAGIC_ROD',
+    'FIVERUPEES': 'RUPEES_20',
+    'HEARTCONTAINER': 'HEART_CONTAINER',
+    'HEARTCONTAINERREFILL': 'HEART_CONTAINER',
+    'HEARTPIECE': 'HEART_PIECE',
+    'HIJUMP': 'FEATHER',
+    'ICEROD': 'MAGIC_ROD',
+    'KEYCT': 'KEY',
+    'KEYDP': 'KEY',
+    'KEYGT': 'KEY',
+    'KEYHC': 'KEY',
+    'KEYIP': 'KEY',
+    'KEYMM': 'KEY',
+    'KEYPD': 'KEY',
+    'KEYSP': 'KEY',
+    'KEYSW': 'KEY',
+    'KEYTH': 'KEY',
+    'KEYTR': 'KEY',
+    'KEYTT': 'KEY',
+    'MAPDP': 'MAP',
+    'MAPEP': 'MAP',
+    'MAPGT': 'MAP',
+    'MAPHC': 'MAP',
+    'MAPIP': 'MAP',
+    'MAPMM': 'MAP',
+    'MAPPD': 'MAP',
+    'MAPSP': 'MAP',
+    'MAPSW': 'MAP',
+    'MAPTH': 'MAP',
+    'MAPTR': 'MAP',
+    'MAPTT': 'MAP',
+    'ONEHUNDREDRUPEES': 'RUPEES_100',
+    'ONERUPEE': 'RUPEES_20',
+    'POWERBOMB': 'BOMB',
+    'PROGRESSIVEGLOVE': 'POWER_BRACELET',
+    'PROGRESSIVESHIELD': 'SHIELD',
+    'PROGRESSIVESWORD': 'SWORD',
+    'PROGRESSIVETUNIC': 'RED_TUNIC',
+    'REDBOOMERANG': 'BOOMERANG',
+    'SILVERARROWS': 'SINGLE_ARROW',
+    'SMMAPBRINSTAR': 'MAP',
+    'SMMAPLOWERNORFAIR': 'MAP',
+    'SMMAPMARIDIA': 'MAP',
+    'SMMAPWRECKEDSHIP': 'MAP',
+    'SOMARIA': 'MAGIC_ROD',
+    'SPACEJUMP': 'FEATHER',
+    'SPEEDBOOSTER': 'PEGASUS_BOOTS',
+    'TENARROWS': 'ARROWS_10',
+    'THREEBOMBS': 'BOMB',
+    'THREEHUNDREDRUPEES': 'RUPEES_500',
+    'TWENTYRUPEES': 'RUPEES_20',
+    'TWENTYRUPEES2': 'RUPEES_20',
+    'VARIA': 'RED_TUNIC',
+
+    # Sonic Adventure 2 Battle
+    'CHAOS EMERALD': 'PIECE_OF_POWER',
+
+    # Super Mario World
+    'P-BALLOON': 'FEATHER',
+
+    # TUNIC
+    "AURA'S GEM": 'SHIELD',  # card that enhances the shield
+    'DUSTY': 'TRADING_ITEM_BROOM',  # a broom
+    'HERO RELIC - HP': 'TRADING_ITEM_HIBISCUS',
+    'HERO RELIC - MP': 'TOADSTOOL',
+    'HERO RELIC - SP': 'FEATHER',
+    'HP BERRY': 'GUARDIAN_ACORN',
+    'HP OFFERING': 'TRADING_ITEM_HIBISCUS',  # a flower
```

Comment:
```suggestion
    'HP OFFERING': 'TRADING_ITEM_HIBISCUS',  # a flower
    'LUCKY UP': 'HEART_CONTAINER',  # card with a heart on it
```
Just another one I thought of o: